querybook/webapp/components/ListDataDoc/ListDataDoc.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Dispatch, IStoreState } from 'redux/store/types';
+import { FullHeight } from 'ui/FullHeight/FullHeight';
+import { Table } from 'ui/Table/Table';
+import * as dataDocActions from 'redux/dataDoc/action';
+import { TaskStatusIcon } from 'components/Task/TaskStatusIcon';
+import { NextRun } from './components/NextRun';
+import { ActiveButtons } from './components/ActiveButtons';
+import { DataDocsName } from './components/DataDocsName';
+import { DataDocSchedule } from './components/DataDocSchedule';
+import { LastRecord } from './components/LastRecord';
+import { LastRun } from './components/LastRun';
+import { setCollapsed } from 'redux/environment/action';
+
+import './ListDataDoc.scss';
+
+const ListDataDoc: React.FC = () => {
+    const dispatch: Dispatch = useDispatch();
+    const env = useSelector((state: IStoreState) => {
+        const { currentEnvironmentId, environmentById } = state.environment;
+        return environmentById[currentEnvironmentId];
+    });
+    const dataDocs = useSelector(
+        (state: IStoreState) => state.dataDoc.dataDocWithSchema
+    );
+    useEffect(() => {
+        dispatch(setCollapsed(true));
+
+        return () => {
+            dispatch(setCollapsed(false));
```

Comment:
this would cause the sidebar to uncollapse even if it is collapsed when going to this UI